static/app/views/metrics/layout.tsx

```diff
@@ -145,26 +144,7 @@ export const MetricsLayout = memo(() => {
       <Layout.Body>
         <FloatingFeedbackWidget />
         <Layout.Main fullWidth>
-          {isEmptyStateDismissed && !hasSentCustomMetrics && (
-            <Banner
-              title={t('Custom Metrics')}
-              subtitle={t(
-                "Track your system's behaviour and profit from the same powerful features as you do with errors, like alerting and dashboards."
-              )}
-              backgroundComponent={<BackgroundSpace />}
-              dismissKey="metrics"
-            >
-              <Button
-                size={isSmallBanner ? 'xs' : undefined}
-                translucentBorder
-                onClick={() => addCustomMetric('banner')}
-              >
-                {t('Set Up')}
-              </Button>
-            </Banner>
-          )}
```

Comment:
we are removing this banner too as we no longer want to encourage users to send custom metrics